lib/constants/whitelabel-providers.ts

```diff
@@ -0,0 +1,65 @@
+import { ProfileMenuIcons } from '@/components/navigation/Icons';
+
+const env = process.env.OC_ENV || process.env.NODE_ENV || 'development';
+
+type WhitelabelProvider = {
+  slug: string;
+  domain: string;
+  logo: string;
+  name: string;
+  /** List of links to display in our Menus. If the link contains an Icon, we'll also render it in the ProfileMenu */
+  links?: {
+    label: string;
+    href: string;
+    /** If provided, link is also rendered in ProfileMenu */
+    icon?: string;
+  }[];
+};
+
+const WHITELABEL_PROVIDERS: WhitelabelProvider[] = [];
```

Comment:
Can we add something for oficollective.com that is available in production?

Reply:
With a SVG logo

Reply:
Yes, just pushed the pending updates.